### DISCUSSION TITLE
Oqtane Support WASI

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#2240] [General] [answered]
Hi,

Oqtane can support WASI?

Thanks!

**Comment by sbwalker** [comment, marked as answer]
@Sharafudeen Oqtane relies on Blazor... and Blazor currently does not support WASI (ie. Wasm on the Server). Steve Sanderson has done some presentations recently which include early experiments of Blazor support for WASI. However I do not believe they are intended for .NET 7 as .NET 7 is focused on Blazor Hybrid and MAUI.
